docs/fr/docs/advanced/additional-responses.md

```diff
@@ -0,0 +1,240 @@
+
+```JSON hl_lines="4-16"
+{
+    "components": {
+        "schemas": {
+            "Message": {
+                "title": "Message",
+                "required": [
+                    "message"
+                ],
+                "type": "objet",
+                "Propriétés": {
+                    "message": {
+                        "title": "Message",
+                        "type": "chaîne"
+                    }
+                }
+            },
+            "Article": {
+                "title": "Article",
+                "required": [
+                    "identifiant",
+                    "évaluer"
+                ],
+                "type": "objet",
+                "properties": {
+                    "identifiant": {
+                        "title": "Identifiant",
+                        "type": "chaîne"
+                    },
+                    "value": {
+                        "title": "Valeur",
+                        "type": "chaîne"
+                    }
+                }
+            },
+            "Erreur de validation": {
+                "title": "Erreur de validation",
+                "required": [
+                    "loc",
+                    "messages",
+                    "taper"
+                ],
+                "type": "objet",
+                "properties": {
+                    "loc": {
+                        "title": "Emplacement",
+                        "type": "tableau",
+                        "items": {
+                            "type": "chaîne"
+                        }
+                    },
+                    "message": {
+                        "titre": "Message",
+                        "type": "chaîne"
+                    },
+                    "taper": {
+                        "title": "Type d'erreur",
+                        "type": "chaîne"
+                    }
+                }
+            },
+            "Erreur de validation HTTP": {
+                "title": "Erreur de validation HTTP",
+                "type": "objet",
+                "properties": {
+                    "detail": {
+                        "title": "Détail",
+                        "type": "tableau",
+                        "items": {
+                            "$ref": "#/composants/schémas/ValidationError"
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+## Types de médias supplémentaires pour la réponse principale
+
+Vous pouvez utiliser ce même paramètre `responses` pour ajouter différents types de médias pour la même réponse principale.
+
+Par exemple, vous pouvez ajouter un type de média supplémentaire `image/png`, en déclarant que votre *opération de chemin* peut renvoyer un objet JSON (avec le type de média `application/json`) ou une image PNG :
+
+```Python hl_lines="19-24 28"
+{!../../../docs_src/additional_responses/tutorial002.py!}
+```
+
+!!! Remarque
```

Comment:
Not sure the panel rendering will work translating it
Did you generate the documentation locally to see the result? 

Applicable to others `!!! note` and `!!! info`

Reply:
yes, and it's look great : https://6346f37f0775e9008d2e2df2--fastapi.netlify.app/fr/advanced/additional-responses/